src/components/common/LabeledSelect.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react'
+import styled from 'styled-components'
+import Select from 'react-select'
+import find from "lodash/find"
+
+const getStringValues = (value) => {
+    return value == null ? [] : value.map((item) => item.value);
+}
+
+const LabeledSelect = ({label, value, options, onChange, ...rest}) => (
```

Comment:
add isMulti if you're using it and add a default value

Reply:
done, no default required, if not set - not Multi

Reply:
well it's a boolean, better to default to false